public/app/features/alerting/unified/RedirectToRuleViewer.tsx

```diff
@@ -76,7 +78,8 @@ export function RedirectToRuleViewer(): JSX.Element | null {
 
   if (rules.length === 1) {
     const [rule] = rules;
-    return <Redirect to={createViewLink(rulesSource, rule, '/alerting/list')} />;
+    const to = createViewLink(rulesSource, rule, '/alerting/list').replace(subUrl, '');
```

Comment:
Another weird `react-router` quirk – passing `to={"/foo"}` in an URL that already has a path like `http://localhost:3000/foo/alerting` becomes `http://localhost:3000/foo/foo/alerting` 🤷‍♂️